run-tests.pl

```diff
@@ -34,17 +35,22 @@
       : undef;
 });
 
+my $plugin_dir = $ENV{"SYTEST_PLUGINS"} || "plugins"; # Read plugin dir from env var SYTEST_PLUGINS or fallback to plugins
+my @plugins = grep { -d } glob(catdir($plugin_dir, "*", "*")); # Read all plugins/<author>/<plugin>
```

Comment:
I'm curious about this structure: why so many directories?

Reply:
So you could use github as a the plugin repository. We could omit the <author> part and keep the sturcture more flat. That would need some changes to the downloading of plugins.

Reply:
I think I'd prefer that.